webcompat/static/js/lib/issues.js

```diff
@@ -395,29 +400,9 @@ issues.MainView = Backbone.View.extend(
       }
     },
 
-    getRemainingComments: function (count) {
```

Comment:
I should probably leave this in here, but it will need to be re-written (because `this.comments.fetchPage` no longer exists)... not sure.